ui/src/views/404/NavigationItems/nav-items.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
As discussed, let's make this a JS module rather than a JSON file, to allow us to add comments, take advantage of Flow, etc